client/build.sbt

```diff
@@ -84,12 +84,11 @@ val CapiModelsVersion = "11.50"
 
 libraryDependencies ++= Seq(
   "com.gu" %% "content-api-models-scala" % CapiModelsVersion,
-  "com.squareup.okhttp3" % "okhttp" % "3.9.0",
+  "com.squareup.okhttp3" % "okhttp" % "3.9.1",
   "org.slf4j" % "slf4j-api" % "1.7.25",
-  "org.scalatest" %% "scalatest" % "3.0.1" % "test",
   "com.google.guava" % "guava" % "19.0" % "test",
-  "org.mockito" % "mockito-all" % "1.9.0" % "test"
-
+  "org.scalatest" %% "scalatest" % "3.0.5" % "test" exclude("org.mockito", "mockito-core"),
```

Comment:
The exclude isn't really important but I thought it's a good idea since we're using `mockito-all` on the following line.
(`mockito-all` is a single JAR, not one that depends on `core` and the others as some libraries do.)

Reply:
good catch